java/kotlin-extractor2/src/main/kotlin/entities/Expression.kt

```diff
@@ -297,37 +298,83 @@ private fun KotlinFileExtractor.extractBinaryExpression(
     val op = expression.operationToken
     val target = expression.resolveCallTarget()?.symbol
```

Comment:
What's the plan for `target == null`?

If we're just going to emit an `errorExpr`, and perhaps add some more info in a supplementary table, then can we handle that case first, and make `isNumericWithName` and `isBinaryPlus` take a non-null KaFunctionSymbol and not have to handle the `null` case?

Reply:
There are cases below where target is null, for example in case of `===`.